[BD-5212]:Update behaviour on link shortening when used with universal link (#10611)
braze-inc/braze-docs: _docs/_user_guide/brazeai/recommendations.md

### Trending item {#trending}

> The "Trending" recommendation model features items that had the most positive momentum when it comes to recent user interactions. We calculate this with a weighted analysis of about 10 weeks of event history, with the heaviest weighting applied to the most recent, approximate 2 weeks. To prevent small fluctuations from affecting the recommendation quality, we apply an activity threshold and statistical smoothing techniques.

Reviewer: ```suggestion
The "Trending" recommendation model features items that have shown the most positive momentum in recent user interactions. We calculate this using a weighted analysis of approximately 10 weeks of event history, with the heaviest weighting applied to the most recent approximate 2 weeks. To prevent small fluctuations from affecting the recommendation quality, we apply an activity threshold and statistical smoothing techniques.
```
